src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(.\fonts\Avenir-Book.ttf);
+  font-weight: 300;
```

Comment:
add font-style: normal;

Reply:
can't find the box-shadow for input:focus in Figma, so I used the same as for input:hover